docs/libraries.md

```diff
@@ -14,16 +14,15 @@ We have a repository of officially maintained language bindings, as well as many
 ## Unified Libraries
 
 Our official "unified" [language binding repository](http://github.com/ev3dev/ev3dev-lang)
-currently has support for C++, Lua, Node.js, and Python.
-These libraries are all built around a single
-[API specification](http://ev3dev-lang.readthedocs.org/en/latest/spec.html)
-that ensures that interface is almost identical for each, and they are being updated and
-enhanced regularly.
+supports multiple implementations of the ev3dev API in a variety of languages.
 
 * Unified bindings:
     * [C++](https://github.com/ddemidov/ev3dev-lang-cpp)
     * [Node.js](https://github.com/wasabifan/ev3dev-lang-js)
     * [Python](https://github.com/rhempel/ev3dev-lang-python)
+    * Java
+      * [@mob41's ev3dev-lang-java](https://github.com/mob41/ev3dev-lang-java), which follows closely to the format of the other libraries.
+      * [@jabrena's ev3dev-lang-java](https://github.com/ev3dev-lang-java/ev3dev-lang-java), an interface more similar to that of LeJOS.
```

Comment:
I put it there originally, but then moved it because I was annoyed that there was something called "ev3dev-lang" in the other section and was too lazy to deal with that :wink: I guess I will move it back down and just ignore the naming for now.